src/util/metadata/side_metadata/global.rs

```diff
@@ -536,12 +536,12 @@ impl SideMetadataSpec {
                 let meta_addr = address_to_meta_address(self, data_addr);
                 if self.log_num_of_bits < 3 {
                     let lshift = meta_byte_lshift(self, data_addr);
-                    let mask = meta_byte_mask(self) << lshift;
+                    let mask = meta_byte_mask(self);
```

Comment:
I think your code on side metadata is correct. The difference is that the mask returned from `meta_byte_mask()` is not shifted to high order bits, but the mask returned from `get_shift_and_mask_for_bits()` is shifted to high order bits. This is unfortunately confusing. Unless you would like to change one of the methods to make them consistent, I would suggest at least adding some comments to those two methods to make it clear.

Reply:
I change it back so that "mask" is always shifted left to the bits that contains useful data.  That will be consistent with other methods.